Makefile

```diff
@@ -57,11 +62,17 @@ run-local: install-apis ## Install CRDs and run provisioner locally
 	kubectl create namespace rukpak-system
 	$(Q)go run provisioner/k8s/main.go
 
+deploy: install-apis ## Deploy the operator to the current cluster
+	kubectl apply -f provisioner/k8s/manifests
+
+run: build-local-container load-image deploy ## Build image and run operator in-cluster
+
 ## --------------------------------------
-## Building
+## Build and Load
```

Comment:
We're missing the `.PHONY: build bin/k8s bin/unpack` that was lost during the rebase.

Reply:
Whoops - had a stale diff. Disregard.